packages/core/src/ComponentsDependencies.ts

```
@@ -11,6 +11,7 @@
   private static _invDependenciesMap = new Map<ComponentConstructor, ComponentConstructor[]>();
 
   static _dependenciesMap = new Map<ComponentConstructor, DependentInfo>();
+  static _inheritedMap = new Map<ComponentConstructor, Boolean>();
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)